src/test/unit/TestSortedDoublyLLUpdateKey.sol

```diff
@@ -1,33 +1,33 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.9;
 
-import "./mocks/SortedDoublyLLFixture.sol";
-import "./helpers/RevertProxy.sol";
-import "./helpers/truffle/Assert.sol";
+import "ds-test/test.sol";
+import "contracts/test/mocks/SortedDoublyLLFixture.sol";
+import "../helpers/RevertProxy.sol";
+import "../interfaces/ICheatCodes.sol";
+
+contract TestSortedDoublyLLUpdateKey is DSTest {
+    ICheatCodes public constant CHEATS = ICheatCodes(HEVM_ADDRESS);
 
-contract TestSortedDoublyLLUpdateKey {
     address[] ids = [address(1), address(2), address(3), address(4), address(5), address(6)];
     uint256[] keys = [uint256(13), uint256(11), uint256(9), uint256(7), uint256(5), uint256(3)];
 
     SortedDoublyLLFixture fixture;
     RevertProxy proxy;
 
-    function beforeAll() public {
+    function setUp() public {
         proxy = new RevertProxy();
```

Comment:
Foundry doesn't seem to have a `beforeAll` equivalent. I'm not sure if it's safe to execute `RevertProxy` before each test. We could wrap this into an `if` block if it is not.

(Same of the other occurrences of `beforeAll`)

Reply:
yes, it shouldn't be an issue since the internal state of `RevertProxy` is not meant to be shared among different test cases - but I agree that the lack of some lifecycle methods in the current version of foundry is good to keep in mind 

> Ah actually it would be great to add additional tests for the reduced coverage - https://coveralls.io/builds/50828602 Should be small only

The only reason why the coverage dropped is because the package `solidity-coverage` is not currently taking into account the foundry tests - unlike the previous truffle-based tests run via `[runSolidityTest.js` ](https://github.com/livepeer/protocol/blob/confluence/test/unit/helpers/runSolidityTest.js). Foundry has recently added a `coverage` feature in one of its recent 2.x releases but we cannot upgrade to it right away due to [this](https://github.com/foundry-rs/foundry/issues) issue in the way foundry 2.x handles L2 networks: once the issue will be addressed I think we can fix the 'fake' coverage drop.